C/src/TaskSetGenerator/gui.py

```diff
@@ -0,0 +1,223 @@
+import tkinter as tk
```

Comment:
Missing documentation and author info.

Reply:
Thank you! I will add documentation and author info